topics/proteomics/tutorials/neoantigen-variant-annotation/tutorial.md

```diff
@@ -340,7 +352,7 @@ The **Cut Columns** tool is used to extract only the fields required for downstr
 
 ## Editing SAV Annotation Columns Using Column Regex Find and Replace
 
-The **Column Regex Find and Replace** tool is used in this step to clean and standardize SAV (Single Amino-acid Variant) annotation fields. SAV entries often begin with prefixes such as `SNV_` or `INDEL_`, which must be reformatted to a consistent delimiter-based structure (`SNV|`, `INDEL|`) to be compatible with downstream peptide-annotation tools such as PepPointer. This formatting step ensures that SAV annotations can later be merged accurately with assembly-derived peptides, despite their inherently different annotation structures.
+The **Column Regex Find and Replace** tool is used in this step to clean and standardize SAV (Single Amino-acid Variant) annotation fields. SAV entries often begin with prefixes such as `SNV_` or `INDEL_`, which must be reformatted to a consistent delimiter-based structure (`SNV|`, `INDEL|`) to be compatible with downstream peptide-annotation tools such as PepPointer.
```

Comment:
The **Column Regex Find and Replace** tool is used in this step to clean and standardize SAV (Single Amino-acid Variant) annotation fields. SAV entries often begin with prefixes such as `SAV_` or `INDEL_`, which must be reformatted to a consistent delimiter-based structure (`SAV|`, `INDEL|`) to be compatible with downstream peptide-annotation tools such as PepPointer.